features/frontend/product_reviews.feature

```diff
@@ -0,0 +1,87 @@
+@products
+Feature: Products
+  In order to know and pick the products
+  As a visitor
+  I want to be able to browse products
```

Comment:
I want to browse and post product reviews

These are old feature description, so I'd avoid spending too much time here, this feature file should be splitted into 2, reading and writing reviews or something like that, but let's leave it as it is, just please correct these obvious mistakes.